app/helpers/application_helper.rb

```diff
@@ -1,2 +1,9 @@
 module ApplicationHelper
+  def time(date)
+    date.strftime('%H:%M')
+  end
+
+  def date(date)
```

Comment:
```suggestion
  def format_date(date)
```

Won't be more readable for potential readers?